src/sass/mixins/_Icon.Mixins.scss

```diff
@@ -7,62 +7,62 @@
 
 // Base icon definition
 @mixin ms-Icon {
-  $icon-font-family: 'FabricMDL2Icons';
```

Comment:
I had originally overwritten this part above the icons, then copied and pasted it back in. Don't know why the diff went so crazy, it's the same code. If it's too hard to read, I can try to fix it.

Reply:
Probably just a whitespace issue? You could try resetting this file back to master, then pasting the new icon definitions back in. But I'm not overly concerned if the code is the same. :)

Reply:
Good call, it was totally whitespace! The odd thing is that the whitespace issue doesn't appear in my local diff. Might need to get an extension to help with that...